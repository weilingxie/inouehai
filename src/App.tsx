
import React from 'react'
import Setup from './Setup'
import MatchSheet from './components/MatchSheet'
import Store from './store/Store'
import './App.css';

function App() {
  return (
    <div className="App">
    <Store>
      <div className="container">
        <div className="row align-items-center justify-content-center">          
          <div className="col-6">
            <h1>Inoue Hai Kihon Competition</h1>            
            <MatchSheet />
          </div> 
        </div>
      </div>
    </Store>
    </div>
  );
}

export default App;
