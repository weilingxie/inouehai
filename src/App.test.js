import React from 'react';
import App from './App';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';


describe('App',()=>{
    test('render without crashing', () => {
    const wrapper =  shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});

