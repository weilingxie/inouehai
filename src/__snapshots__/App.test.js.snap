// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render without crashing 1`] = `
<div
  className="App"
>
  <div
    className="container"
  >
    <div
      className="row align-items-center justify-content-center"
    >
      <div
        className="sm-12 md-8 lg-6"
      >
        <h1>
          Inoue Hai Kihon Competition
        </h1>
        <ScoreBoard
          score={
            Array [
              Object {
                "name": "A",
                "score": 0,
              },
              Object {
                "name": "B",
                "score": 0,
              },
              Object {
                "name": "C",
                "score": 0,
              },
              Object {
                "name": "D",
                "score": 0,
              },
              Object {
                "name": "E",
                "score": 0,
              },
            ]
          }
        />
        <hr />
        <MatchSheet
          changeColor={[Function]}
          fights={
            Array [
              Object {
                "id": 1,
                "kata": "",
                "kihon": "",
                "suburi": "",
                "team1": Object {
                  "color": "red",
                  "name": "A",
                },
                "team2": Object {
                  "color": "white",
                  "name": "B",
                },
              },
              Object {
                "id": 2,
                "kata": "",
                "kihon": "",
                "suburi": "",
                "team1": Object {
                  "color": "red",
                  "name": "A",
                },
                "team2": Object {
                  "color": "white",
                  "name": "C",
                },
              },
              Object {
                "id": 3,
                "kata": "",
                "kihon": "",
                "suburi": "",
                "team1": Object {
                  "color": "red",
                  "name": "D",
                },
                "team2": Object {
                  "color": "white",
                  "name": "C",
                },
              },
              Object {
                "id": 4,
                "kata": "",
                "kihon": "",
                "suburi": "",
                "team1": Object {
                  "color": "red",
                  "name": "D",
                },
                "team2": Object {
                  "color": "white",
                  "name": "E",
                },
              },
              Object {
                "id": 5,
                "kata": "",
                "kihon": "",
                "suburi": "",
                "team1": Object {
                  "color": "red",
                  "name": "B",
                },
                "team2": Object {
                  "color": "white",
                  "name": "E",
                },
              },
            ]
          }
          resetResult={[Function]}
          setResult={[Function]}
        />
      </div>
    </div>
  </div>
</div>
`;
